uk.co.compucorp.civicrm.hrcore/backstop_data/casper_scripts/contact-summary/actions/show-user-account-delete-confirmation-dialog.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+var page = require('../../../page-objects/contact-summary');
+
+module.exports = function (casper) {
+  page.init(casper).deleteUserAccount();
```

Comment:
this
```js
page.init(casper).openContactActionMenu().deleteUserAccount();
```
should be the correct approach, given that in fact in the `deleteUserAccount` method you are replicating the logic of `openContactActionMenu`